zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -123,8 +123,8 @@ if __name__ == "__main__":
         try:
             sb = SlackBridge(config)
 
-            zp = mp.Process(target=sb.zulip_client.call_on_each_message, args=(sb.zulip_to_slack(),))
-            sp = mp.Process(target=sb.run_slack_listener, args=())
+            zp = threading.Thread(target=sb.zulip_client.call_on_each_message, args=(sb.zulip_to_slack(),))
+            sp = threading.Thread(target=sb.run_slack_listener, args=())
```

Comment:
I'm not sure about the previous `Process` case, but my understanding is that exceptions cannot be propagated across threads; are these targets safe from that? What is the Exception block below expected to handle?

Reply:
I can't make a definitive answer to your question without testing how the code behaves.